pkgs/servers/web-apps/snipe-it/composer-env.nix

```
@@ -0,0 +1,239 @@
+# This file originates from composer2nix
+
+{ stdenv, lib, writeTextFile, fetchurl, php, unzip, phpPackages }:
+
+let
+  inherit (phpPackages) composer;
+  buildZipPackage = { name, src }:
+    stdenv.mkDerivation {
+      inherit name src;
+      buildInputs = [ unzip ];
```

```suggestion
      nativeBuildInputs = [ unzip ];
```

otherwise RIP cross compilation.